### PR TITLE
PLAYER-2973

### PR DESCRIPTION
**Pull request by AlinaVoronkovaEpam**

### CODE REVIEW

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java [hunk 1]
```diff
@@ -318,4 +320,20 @@ public static WritableMap buildVRParams(boolean vrContent, boolean isStereoSuppo
...
+      ArrayList<AudioTrack> audioTracks = player.getAvailableAudioTracks();
+      WritableArray audioTracksTitles = Arguments.createArray();
+
+      for (AudioTrack track : audioTracks) {
+        audioTracksTitles.pushString(track.getTrackTitle());
+      }
+
+      params.putBoolean("multiAudioEnabled", true);
+      params.putArray("audioTracksTitles", audioTracksTitles);
+      }
```
Comment (atopilskii) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java]: extra space?

Comment (AlinaVoronkovaEpam) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java]: thanks. fixed.

**Review by atopilskii — APPROVED** [review]

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java [hunk 2]
```diff
@@ -318,4 +320,20 @@ public static WritableMap buildVRParams(boolean vrContent, boolean isStereoSuppo
     params.putBoolean("stereoSupported", vrContent && isStereoSupported);
     return params;
   }
+
+  public static WritableMap buildMultiAudioParams(OoyalaPlayer player) {
```
Comment (IvanSakharovskii) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java]: Can we do it without `ooyalaPlayer`?

Comment (AlinaVoronkovaEpam) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java]: done.

**Review by IvanSakharovskii — APPROVED** [review]

**Pull request merged**